domenetjenester/beregningsgrunnlag/src/main/java/no/nav/foreldrepenger/domene/prosess/BeregningKalkulus.java

```diff
@@ -136,10 +136,10 @@ public void kopier(BehandlingReferanse revurdering, BehandlingReferanse original
             LOG.info("Kobling for behandlingUuid {} finnes ikke, oppretter", revurdering.behandlingUuid());
             return koblingRepository.opprettKoblingFraOriginal(revurdering, originalKobling.get());
         });
-        if (!tilstand.equals(BeregningsgrunnlagTilstand.FASTSATT)) {
-            throw new IllegalStateException("Støtter ikke kopiering av grunnlag som ikke er fastsatt!");
+        if (!tilstand.equals(BeregningsgrunnlagTilstand.FASTSATT) && !tilstand.equals(BeregningsgrunnlagTilstand.FORESLÅTT)) {
```

Comment:
Hvordan er kopieringssemantikk for FASTSATT vs FORESLÅTT. Regner med FASTSATT vil ha til-og-med - men hva er behovet for FORESLÅTT